id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -16,22 +16,22 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
 
   implicit val session = AutoSession(db.settingsProvider)
 
-  def findSourceIdInDb(miroId: String): Future[Option[Identifier]] =
+  def lookupMiroID(miroID: String): Future[Option[Identifier]] =
```

Comment:
But really I just didn’t want to continue down a path where Miro ID is a “special” identifier – and exactly what we need when Sierra/CALM enter will determine what the final structure looks like.